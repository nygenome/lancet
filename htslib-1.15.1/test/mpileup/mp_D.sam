@HD	VN:1.6	SO:coordinate
@SQ	SN:z	LN:13
@CO	
@CO	Copyright (c) 2018 Genome Research Ltd.
@CO	
@CO	Permission is hereby granted, free of charge, to any person obtaining
@CO	a copy of this software and associated documentation files (the
@CO	"Software"), to deal in the Software without restriction, including
@CO	without limitation the rights to use, copy, modify, merge, publish,
@CO	distribute, sublicense, and/or sell copies of the Software, and to
@CO	permit persons to whom the Software is furnished to do so, subject
@CO	to the following conditions:
@CO	
@CO	The above copyright notice and this permission notice shall be included
@CO	in all copies or substantial portions of the Software.
@CO	
@CO	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
@CO	OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
@CO	MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
@CO	IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
@CO	CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
@CO	TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
@CO	SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
@CO	
@CO	              1
@CO	     1234567890123 Depadded base numbering
@CO	ref  TAGCTTAGCAGGT
@CO	
@CO	s1    AGCTTAGCAGG  11M
@CO	s2    AGCT***CAGG  4M 3D 4M
@CO	s3    **CTTAGCA**  2D 7M 2D
@CO	qual  01234567890
@CO	
s1	0	z	2	0	11M	*	0	0	AGCTTAGCAGG	01234567890
s2	0	z	2	0	4M3D4M	*	0	0	AGCTCAGG	01237890
s3	0	z	2	0	2D7M2D	*	0	0	CTTAGCA	2345678
