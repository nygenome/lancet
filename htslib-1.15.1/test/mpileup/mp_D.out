z	2	3	^!A^!A^!*
z	3	3	GG*
z	4	3	CCC
z	5	3	TT-3()T
z	6	3	T*T
z	7	3	A*A
z	8	3	G*G
z	9	3	CCC
z	10	3	AAA-2()
z	11	3	GG*
z	12	3	G$G$*$
